scss/tech.scss

```diff
@@ -0,0 +1,43 @@
+@import 'variables';
+
+.tech {
+  background: $color-green;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  height: 300px;
+  justify-content: space-between;
+  margin: 15px 20px;
+  padding: 20px;
+  width: 200px;
+
+  &::before {
+    content: '';
+    border-radius: 5px;
+    border: 2px solid $color-green;
+    height: 296px;
+    width: 196px;
+    position: absolute;
+    z-index: -1;
+    top: 118px;
```

Comment:
You should add `position: relative` in parent so this elements positions absolute to it